{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN invenio collect -v  && \
     invenio webpack install --unsafe && \
     invenio webpack build
 
-ENV INVENIO_THEME_LOGO='images/logo.svg'
-ADD ./{{cookiecutter.project_shortname}}/static/images/logo.svg ${INVENIO_INSTANCE_PATH}/static/images/logo.svg
+ENV INVENIO_THEME_LOGO="images/logo.svg"
+ADD ./static/images/logo.svg /opt/invenio/var/instance/static/images/logo.svg
```

Comment:
minor (can be shelved): Sorry if that wasn't clear in the last comment: should we use "inveniordm" (or even stay with `${INVENIO_INSTANCE_PATH}`) since `inveniordm` is the application versus `invenio` the framework?

Reply:
I would vote to stay with `${INVENIO_INSTANCE_PATH}` since we are based on Invenio and otherwise it might bring incompatible changes arising from `docker-invenio`.